src/components/MenuSelect.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import Template from './Template';
+import autoHideContainerHOC from '../decorators/autoHideContainer.js';
+import headerFooterHOC from '../decorators/headerFooter.js';
+
+class MenuSelect extends Component {
+  static propTypes = {
+    cssClasses: PropTypes.shape({
+      select: PropTypes.string,
+      option: PropTypes.string,
+    }),
+    items: PropTypes.array.isRequired,
+    refine: PropTypes.func.isRequired,
+    templateProps: PropTypes.object.isRequired,
+  };
+
+  get selectValue() {
```

Comment:
This is not very idiomatic with the rest of the codebase. Do we want to move forward with that notation?

Reply:
Let's use the JavaScript features we are offered no? (When it makes sense).
I like JavaScript getters, and here it makes sense because I use this value more than once.